pelicun/model/damage_model.py

```diff
@@ -1296,6 +1304,74 @@ def map_ds(values: np.ndarray, offset: int) -> np.ndarray:
 
             return ds_id
 
+        def parse_scaling_specification(scaling_specification: dict) -> dict:  # noqa: C901
+            """
+            Parse and validate the scaling specification.
+
+            Parameters
+            ----------
+            scaling_specification : dict
+                The scaling specification to be parsed and validated.
+
+            Returns
+            -------
+            dict
+                The parsed and validated scaling specification.
+
+            Raises
+            ------
+            ValueError
+                If the scaling specification is invalid.
+            TypeError
+                If the type of an entry is invalid.
+            """
+            # if there are contents, ensure they are valid.
+            # See docstring for an example of what is expected.
+            parsed_scaling_specification = {}
+            # validate contents
+            for key, value in scaling_specification.items():
+                # loop through limit states
+                if key not in parsed_scaling_specification:
+                    parsed_scaling_specification[key] = {}
+                if 'ALL' in value:
+                    if len(value) > 1:
+                        msg = (
+                            f'Invalid entry in scaling_specification: '
+                            f'{value}. It should only have one entry for `ALL`.'
+                        )
+                        raise ValueError(msg)
```

Comment:
I recommend adding another check here in under an `elif` testing that value startswith `LS`. This way you can provide a potentially useful error message for people who send in something else.